src/parser.ts

```diff
@@ -662,6 +672,165 @@ const TRAVERSE_BUILDER: MarkVisitor<(rhs: TraversalResult | null) => TraversalRe
   },
 }
 
+const SELECTOR_BUILDER: MarkVisitor<null> = {
+  square_bracket(p) {
+    p.process(TRAVERSE_BUILDER)
+    return null
+  },
+
+  slice(p) {
+    p.process(TRAVERSE_BUILDER)
+    return null
+  },
+
+  group(p) {
+    // This should only throw an error until we add support for groups in selectors.
+    throw new Error('Invalid selector syntax')
```

Comment:
It's very easy adding support though:

```suggestion
    p.process(SELECTOR_BUILDER)
```